src/Symfony/Component/CssSelector/ConverterInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\CssSelector;
+
+interface ConverterInterface
+{
+    /**
+     * Translates a CSS expression to its XPath equivalent.
+     *
+     * Optionally, a prefix can be added to the resulting XPath
+     * expression with the $prefix parameter.
+     *
+     * @param string $cssExpr The CSS expression.
+     * @param string $prefix  An optional prefix for the XPath expression.
+     *
+     * @return string
+     *
+     * @api
```

Comment:
Well, we haven't tagged anything as ``@api`` since 2.3 AFAICT. And we still work hard on BC for all these classes.

Btw, the api tag BC rules are not stricter. they are the real rules. the special rules are for non-api classes, where we allow BC breaks, but only some of them. And as we became better at writing BC layers and worse at tagging things as ``@api``, this distinction does not really make sense IMO. We already apply strict BC almost everywhere